content/building/cmake-presets.md

```diff
@@ -4,6 +4,123 @@
+  - Allows you to specify whether to include, or not, support for Application Domains. Default is OFF. More information about this is available in the documentation [here](https://msdn.microsoft.com/en-us/library/cxk374d9(v=vs.90).aspx). **Note that the complete removal of support for this feature is being considered (see issue [here](https://github.com/nanoframework/nf-interpreter/issues/303)).**
+- "NF_FEATURE_WATCHDOG" : "**ON**"
+  - Allows you to define it the hardware watchdog should be disabled.
+   This setting can only be set to OFF for STM32 targets. ESP32 build enables this by default so there is no way to disable it.
+   Default is ON, so the hardware watchdog will be enabled by default.
+- "NF_FEATURE_HAS_CONFIG_BLOCK" : "**OFF**"
+  - Allows the developer to set if the targets platform has configuration block.
+  This requires the the block storage definition and the linker files add support for that.
+  Default is OFF meaning that that the target DOES NOT have configuration block.
+- "NF_PLATFORM_NO_CLR_TRACE" : "**OFF**"
+  - Allows you to define if trace messages and checks are added to CLR or not.
+  These checks are usually valuable when debugging issues within the CLR.
+  Can and should be removed for RTM build flavours.
+  Default is OFF meaning that all the standard trace and checks are added to the CLR.
+- "NF_CLR_NO_IL_INLINE" : "**OFF**"
+  - Allows you to define if CLR will use IL inlining.
+  Default is OFF meaning that CLR will inline IL.
+- "NF_INTEROP_ASSEMBLIES" : [ "Assembly1-Namespace", "Assembly2-Namespace" ]
+  - Lists the name of the Interop assembly(ies) to be added to the build. Leave empty or don't add it if no Interop assembly is to be added.
+- "NF_NETWORKING_SNTP" : "**ON**"
+  - Allows you to specify whether SNTP is enabled. Requires networking feature to be enabled. Default is ON.
+- "NF_SECURITY_MBEDTLS" : "**OFF**"
+  - Enables support for secure sockets using mbedTLS. Default is OFF.
+- "MBEDTLS_SOURCE" : "**OFF**"
+  - Path to an optional local with mbedTLS source files.
+- "API_nanoFramework.Devices.OneWire" : "**OFF**"
+  - Allows you to specify whether support for Devices.OneWire is available to your application. Default is OFF.
+- "API_System.Devices.Dac" : "**OFF**"
+  - Allows you to specify whether DAC functions are available to your application. Default is OFF.
+- "API_System.Math" : "**OFF**"
+  - Allows you to specify whether System.Math support is available to your application. Default is OFF.
+- "API_System.Net" : "**OFF**"
+  - Allows you to specify whether System.Net support is available to your application. Default is OFF.
+- "API_nanoFramework.Device.Can" : "**OFF**"
+  - Allows you to specify whether CAN bus functions are available to your application. Default is OFF.
+- "API_nanoFramework.Devices.OneWire" : "**OFF**"
+  - Allows you to specify whether 1-Wire functions are available to your application. Default is OFF.
+- "API_Windows.Devices.Adc" : "**OFF**"
```

Comment:
```suggestion
- "API_System.Device.Adc" : "**OFF**"
```

Reply:
All those `Windows.Devices` can be safely removed now. Except Storage and Streams.